include/onnxruntime/core/common/logging/macros.h

```diff
@@ -251,6 +252,17 @@
     if (::onnxruntime::logging::vlog_enabled && level <= (logger).VLOGMaxLevel())    \
       LOGF_USER_CATEGORY(logger, VERBOSE, "VLOG" #level, format_str, ##__VA_ARGS__); \
   } while (0)
+#else
+// Disabled in Release builds.
+#define VLOGS(logger, level) \
+  if constexpr (false) LOGS_CATEGORY(logger, VERBOSE, "VLOG" #level)
```

Comment:
>if constexpr (false) [](http://example.com/codeflow?start=2&length=20)

maybe do `if constexpr (false) {} else LOGS(...)`
otherwise we may run into the same issue addressed here:
[https://github.com/microsoft/onnxruntime/pull/7589](https://github.com/microsoft/onnxruntime/pull/7589)

Reply:
Done.